common/app/layout/slices/Container.scala

```diff
@@ -33,6 +33,7 @@ object Container extends GuLogging {
       ("nav/list", NavList),
       ("nav/media-list", NavMediaList),
       ("news/most-popular", MostPopular),
+      ("flexible/special", Dynamic(FlexibleSpecial)),
```

Comment:
There is future work to refactor flexible out of the dynamic class but this will be done in an upcoming PR